moped-editor/src/views/projects/projectsListView/ProjectsListView.js

```diff
@@ -32,15 +31,10 @@ let projectsQuery = new GQLAbstract(PROJECT_LIST_VIEW_QUERY_CONFIG);
  */
 const ProjectsListView = () => {
   const classes = useStyles();
-  const navSearchTerm = useLocation()?.state?.searchTerm;
```

Comment:
I moved this into the `ProjectListViewTable` component since useLocation is used there too and the search term isn't used in this component anymore.